templates/index.html

```diff
@@ -202,6 +203,14 @@ <h1>Image Similarity on Wikimedia Commons</h1>
 			return upload_to_thumbnail(url);
 		}
 
+		var generate_permalink = function() {
+			var result_url = 'https://imagesimilarity.toolforge.org/';
+			if (document.getElementById('image-name').value) {
+				result_url = result_url + '?image_name=' + document.getElementById('image-name').value;
```

Comment:
Could you add a `.replaceAll(' ', '_')` to the image name so there aren't any spaces in the URL? It helps with copying the text fully and where browsers enforce line breaks to have the URL be one fully-connected string of characters.